integrated_channels/sap_success_factors/transmitters/content_metadata.py

```diff
@@ -63,6 +63,15 @@ def transmit(self, create_payload, update_payload, delete_payload, content_updat
                     )
                 )
                 self.client.update_content_metadata(self._serialize_items(chunked_items))
+                LOGGER.info(
+                    generate_formatted_log(
+                        self.enterprise_configuration.channel_code(),
+                        self.enterprise_configuration.enterprise_customer.uuid,
+                        None,
+                        None,
+                        f'Successfully sent {list(chunk.keys())}'
```

Comment:
this will help us audit the job run logs to know with more certainty that specific courses the customer pointed out successfully sent